stable/kanister-operator/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: profile
+    version: 0.1.0
```

Comment:
This can't be templated, correct? What happens when we update the profile chart?

Reply:
from https://github.com/kubernetes/helm/blob/master/docs/helm/helm_dependency.md
``` The 'version' field should contain a semantic version or version range. ```

Reply:
something like  version: =>0.1.0  should work

Reply:
Yeah, we can do that..https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/conventions.md